draft-ietf-scitt-software-use-cases.md

```diff
@@ -347,15 +350,15 @@ There is no standardized way to:
 ## Identify Statements and Updates to Specific Versions of Released Software
 
 Software producers often have multiple and concurrent supported versions of a product.
-The versions may represent major feature or compatibility differentiating releases (1.0, 2.0), or implementations for different Operating System Platforms and Architectures (Linux, Mac, Windows, AMD, ARM, x86, x64).
+The versions may represent major feature or compatibility differentiating releases (1.0, 2.0), or implementations for different Operating System Platforms and their respective Instruction Set Architectures (AMD, ARM, x86, x64 for Linux, Mac, and Windows).
```

Comment:
Minor, non-blocking: we want to be combinatorial and show the problem here I believe, I would word it to make it clear we mean MxN actual "versions" to keep tabs on.